api/src/scripts/populate_db.py

```diff
@@ -123,39 +125,47 @@ def add_entity(entity, priority):
         if self.df is None:
             return
 
-        # Keep a dict (map) of stable_id -> feed so we can reference the feeds when processing the static_reference
+        # Gather all stable IDs
```

Comment:
I'll be a bit of a stickler. I would change the comment:
    # Gather all stable IDs `from the CSV file.`